assets/misc/meilisearch-collection-postman.json

```diff
@@ -2618,7 +2624,7 @@
 						"header": [],
 						"body": {
 							"mode": "raw",
-							"raw": "{\n    \"scoreDetails\": true\n}",
+							"raw": "{\n    \"exportPuffinReports\": true\n}",
```

Comment:
Let's use a more shiny feature like AI instead 😇

Reply:
I sort of didn't want to have vector search here to avoid continuously associating it with experimental/unstable features.

Reply:
In this case, use `metrics` instead. It's way more useful for the users, which is not the case of `exportPuffinReports`